composites/Plugin/Shared/components/Collapsible.js

```diff
@@ -72,7 +74,19 @@ const StyledHeading = wrapInHeading( StyledIconsButton, 2 );
 /**
  * Base collapsible panel. Optionally has a heading around the button.
  *
- * @param {object} props The properties for the component.
+ * @param {object}      props                       The properties for the component.
+ * @param {children}    props.children              The content of the Collapsible.
+ * @param {IconsButton} props.Heading               Heading button. May be wrapped or styled or both.
+ * @param {boolean}     props.isOpen                True displays the children. False means collapsed.
+ * @param {function}    props.onToggle              Function to handle the Heading click event.
+ * @param {string}      props.prefixIcon            Heading icon before the title.
+ * @param {string}      props.prefixIconCollapsed   Prefix icon when in collapsed state.
+ * @param {string}      props.prefixIconColor       CSS color of the prefix icon.
+ * @param {string}      props.suffixIcon            Heading icon after the title.
+ * @param {string}      props.suffixIconColor       CSS color of the suffix icon.
+ * @param {string}      props.suffixIconCollapsed   Suffix icon when in collapsed state.
+ * @param {string}      props.title                 Title for in the Heading.
+ * @param {string}      props.titleScreenReaderText Chance for an extra
```

Comment:
I'm not sure what "Chance for an extra" means for this parameter. Could you clarify the description?